chef_master/source/release_notes_server.rst

```diff
@@ -11,12 +11,48 @@ Chef is a systems and cloud infrastructure automation framework that makes it ea
 
 .. end_tag
 
+What's New in 12.13
+=====================================================
+The following items are new for Chef server 12.13:
+
+* **Supports Red Hat Enterprise Linux 6 on s390x (RHEL6/s390x).**
+* **Disables the Solr4 Admin API/UI by default.**
+* **FIPS runtime flag exposed on RHEL systems.** Setting ``fips true`` and reconfiguring will start the server in FIPS mode.  Packages for other systems will not have the required OpenSSL FIPS module and will fail to start if reconfigured with ``fips true``.
+
+New platform: RHEL6/s390x
+=====================================================
+Support for a new platform was added: Red Hat Enterprise Linux 6 on s390x.
+
+Solr4 Admin API/UI disabled by default
+=====================================================
+With this release, the admin UI of Solr4 has been removed. The underlying API
+has also been disabled. Users that depend on the admin API endpoints can enable
+them via adding:
+
+.. code-block:: ruby
+
+    opscode_solr4['enable_full_admin_api'] = true
+
+to ``chef-server.rb``.
+
+FIPS runtime flag exposed
+=====================================================
+The Chef Server package now exposes a ``fips`` configuration flag in
```

Comment:
We should include this new config flag in the config_rb_server.rst file with the appropriate version annotation.